cura/PrinterOutputDevice.py

```diff
@@ -34,6 +34,12 @@ class ConnectionState(IntEnum):
     busy = 3
     error = 4
 
+class ConnectionType(IntEnum):
+    none = 0
```

Comment:
In most cases of Enums that could find we use UpperCamelCase

Reply:
Fixed in 6bf39a32a941dfc4ef96a3f250527c04d9060541

Reply:
@nallath PascalCase! How dare you rob me of my case name!